core/src/main/java/fj/P2.java

```diff
@@ -1,8 +1,10 @@
 package fj;
 
 import static fj.Function.*;
-
+import static fj.data.optic.PLens.pLens;
```

Comment:
Another reason is that instances for List regarding things like Equal, Show, etc. are in those classes, rather than List.